testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeCheckConstraintCompatibility.java

```diff
@@ -137,6 +137,69 @@ public void testCheckConstraintUpdateCompatibility()
         }
     }
 
+    // TODO: Add DELTA_LAKE_DATABRICKS and DELTA_LAKE_EXCLUDE_73 groups once flakiness on Databricks is resolved
```

Comment:
nit: `Support check constraints for MERGE statement in Delta` -> `Tets check constraints for MERGE statement in Delta`